projects/netgrif-components-core/src/lib/authentication/session/services/session-idle-timer.service.ts

```diff
@@ -0,0 +1,53 @@
+import {Injectable, OnDestroy} from "@angular/core";
+import {interval, Observable, ReplaySubject, Subscription} from "rxjs";
+import {ConfigurationService} from "../../../configuration/configuration.service";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class SessionIdleTimerService implements OnDestroy {
+
+    public static readonly SESSION_TIMEOUTTIME = 900;
```

Comment:
if this is the default time when none is specified in the configuration, it should have a name that mentions this fact